modules/vpc/variables.tf

```diff
@@ -11,3 +11,7 @@ variable "cidr_block" {
 variable "availability_zones" {
   description = "The availability zones to create subnets in"
 }
+
+variable "cluster_name" {
```

Comment:
We currently (at cookpad) have the pattern of many clusters sharing a VPC, perhaps we should make this an array of cluster names?